MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -283,13 +281,14 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsT
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if(numberOfMarkups < 1)
+  if( numberOfMarkups < 1 )
   {
-    if(markupsToModelModuleNode->GetModelNode()!=NULL)
+    if ( markupsToModelModuleNode->GetModelNode() != NULL && markupsToModelModuleNode->GetModelNode()->GetPolyData() != NULL )
     {
+      // TODO: Review this code, see if it is really needed...
```

Comment:
I've submitted a pull request for fixing this "No input data" warning in Slicer core (https://github.com/Slicer/Slicer/pull/563)